examples/reference/panes/LaTeX.ipynb

```diff
@@ -41,11 +41,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "latex = pn.pane.LaTeX('The LaTeX pane supports two delimiters: $LaTeX$ and \\(LaTeX\\)', \n",
+    "latex = pn.pane.LaTeX(r'The LaTeX pane supports two delimiters: $LaTeX$ and \\(LaTeX\\)', \n",
```

Comment:
I added the `r` every where. I believe this will help users. If you see a working example without it, and your example does not work, then the cause is not very clear.